samples/nrf9160/at_client/README.rst

```diff
@@ -0,0 +1,130 @@
+.. _at_client_sample:
+
+nRF9160: AT Client Sample
+#########################
+
+The AT Client sample demonstrates the asynchronous serial communication taking place over UART to the nRF9160 modem. AT client is a stand-alone proxy firmware which enables an external computer or MCU to get access to the nRF9160 modem API. 
+
+
+Overview
+********
+
+The AT Client sample uses AT commands for sending directives to the nRF9160 modem. This facilitates the reading of responses or analyzing of events related to the nRF9160 modem. The commands can be initiated from a terminal or the `LTE Link Monitor`_ which is an application implemented as part of the `nRF Connect for Desktop`_ . 
+
+For more information on the AT commands, see `AT Commands Reference`_ .
+
+Also see :ref:`modem_info_readme` for more information regarding the process of interaction between an LTE application and the modem using the modem information library.
+
+
+Requirements
+************
+
+* The following development kit board:
+
+  * nRF9160 DK board(PCA10090) connected using micro USB cable to the external computer or MCU.
+
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/nrf9160/at_client`
+
+.. include:: /includes/spm.txt
+
+
+Connecting the LTE Link Monitor and the nRF9160 DK Board
+========================================================
+
+After programming the AT Client sample onto the nRF9160 DK board, launch the LTE Link Monitor. Once the nRF9160 DK Board is powered on, click :command:`Select Device` and select the particular board device from the drop down list in the LTE Link Monitor. After this, perform a reset of the nRF9160 DK board by pressing the reset button, which will reboot the board and start the AT Client sample. Initially the command :command:`AT+CFUN?` is automatically sent to the modem, which returns a value 4, indicating that the modem is in the offline mode. After a while, the LTE Link Monitor terminal display shows :command:`AT+CFUN=1` indicating that the modem is in the normal mode.   
+
+Testing
+=======
+
+The AT Client sample can be tested by running the following commands from the LTE Link Monitor terminal:
+
+#. Enter the command :command:`AT+CFUN?` 
+
+   * This command connects to an LTE network and triggers the command :command:`AT+CFUN=1` which will change the modem state to the normal mode.
+   * Enter the command :command:`AT+CFUN?` into the LTE Link Monitor terminal again. The UART/Modem/UICC/LTE/PDN indicators in the LTE Link Monitor side panel turns into green.
```

Comment:
Remove 'into'. @ru-fu  Style: Is future tense used here 'will turn green' or present 'turns green' ?

Reply:
I avoid future tense unless it's absolutely required, so I think "turns green" is fine.
SDK style is actually to write it as action, so "Observe that ... turns green". Not sure if I really like that though. ;)

Reply:
Updated to: The UART/Modem/UICC/LTE/PDN indicators in the LTE Link Monitor side panel turns green.